format/proto/types.proto

```diff
@@ -27,7 +27,7 @@ enum OrchestratorResponseType {
     // The status of a workflow
     WORKFLOW_STATUS = 0;
     // Id of a created workflow
```

Comment:
Comment doesn't seem to match anymore?

Reply:
Workflow_id is used as a response for all create workflow operations. It will be shared, not just for create node workflow.

Reply:
But why is the enum called WORKFLOW_CREATE? I'm lost here.

Reply:
I'm not sure I understand your question. WORKFLOW_CREATE doesn't refer to create node workflow, it refers to any workflow that is created .

Reply:
The comments should probably say that then.

Reply:
Ok, will change it in the remove node PR.